<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdvokat</class>
 <widget class="QWidget" name="DialogAdvokat">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ID Advokat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editIDAdvokat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nama Advokat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNamaAdvokat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Spesialisasi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editSpesialisasi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Lisensi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editLisensi">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cari</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cariAdvokat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelAdvokat">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lisensi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Spesialisasi</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSimpan">
     <property name="text">
      <string>Simpan Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnUbah">
     <property name="text">
      <string>Ubah Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBatal">
     <property name="text">
      <string>Batal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
